scripts/fix-markdown-strings.py

```diff
@@ -22,6 +22,14 @@
     "## If you want to know more": "## ℹ️ If you want to know more",
 }
 
+ADMONITION_TITLE = "These are the key outcomes of this assessment"
+
+QA_STATEMENT_FORMAT = {
+    "l0": "## 📢 Quality assessment statement",
+    "l2": f"```{{admonition}} {ADMONITION_TITLE}",
+    "l3": ":class: note",
+}
+
 
 def fix_legacy_urls(path: Path) -> None:
```

Comment:
If the new functionality is added to the old function, we should revise the name (BTW, I think the name is wrong... I forgot to rename it when I made this function more general)